.eslintrc.js

```diff
@@ -1,15 +1,29 @@
 module.exports = {
-  extends: ['airbnb-base', 'plugin:import/errors', 'prettier'],
-  plugins: ['prettier', 'mocha'],
+  parser: 'babel-eslint',
+  extends: ['airbnb', 'prettier'],
```

Comment:
Do we need to remove `plugin:import/errors` here?

Reply:
I was comparing the eslint config with another project and there I didn't use the `eslint-plugin-import`, so I thought let's remove it. Fun fact: I forgot to remove the dependency which I did just right now. However now I get this error `couldn't find the plugin "eslint-plugin-import".` and I don't know where this error comes from. 

For now I think it's okay to keep it like it is and I'll revisit this some day. It shouldn't harm even the plugin is magically in use.

Reply:
It looks like `eslint-config-airbnb-base` includes `eslint-plugin-import`, so we're still implicitly using it: https://github.com/airbnb/javascript/blob/ad3746672e5bd79f77159c4e2612077651e9af17/packages/eslint-config-airbnb-base/package.json#L56

Reply:
Hold on, I removed `eslint-config-airbnb-base` https://github.com/OctoLinker/browser-extension/pull/386/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2, but I fear I didn't used `yarn remove *`. Will fix this in another PR

Reply:
Looks like everything is correct. `eslint-config-airbnb` includes `eslint-config-airbnb-base` which is the reason why it's still listed in `yarn.lock`. Anyway, let's ignore this for now 😉

Reply:
Yes, I think everything is in order, and there's no issue remaining to ignore :)